.buildkite/commands/upload-code-coverage.sh

```diff
@@ -0,0 +1,8 @@
+curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import
+curl -Os https://uploader.codecov.io/latest/linux/codecov
+curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM
+curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig
+gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
+sha256sum -c codecov.SHA256SUM
+chmod +x codecov
+./codecov -t $CODECOV_TOKEN
```

Comment:
Running [`shellcheck`](https://github.com/koalaman/shellcheck) on this script to lint it suggested me to secure this variable expansion by putting it in quotes:
```
$ shellcheck .buildkite/commands/*            

In .buildkite/commands/upload-code-coverage.sh line 1:
curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import
^-- SC2148 (error): Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.


In .buildkite/commands/upload-code-coverage.sh line 8:
./codecov -t $CODECOV_TOKEN
             ^------------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
./codecov -t "$CODECOV_TOKEN"

For more information:
  https://www.shellcheck.net/wiki/SC2148 -- Tips depend on target shell and y...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

```suggestion
./codecov -t "$CODECOV_TOKEN"
```

Reply:
Thanks! TIL about `shellcheck`